src/test/bind/test_bind.c

```diff
@@ -291,167 +172,83 @@ static int _check_matching_addresses(int fd_server_listen, int fd_server_accept,
      *   + client socket pot != accepted peer ports
```

Comment:
I know you didn't introduce it, but there is a spelling error here: pot -> port.

Reply:
Done